src/cpp/fastdds/utils/QosConverters.hpp

```diff
@@ -0,0 +1,110 @@
+// Copyright 2019 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file QosConverters.h
+ */
+
+
+
+#ifndef _QOS_CONVERTERS_HPP_
+#define _QOS_CONVERTERS_HPP_
+#ifndef DOXYGEN_SHOULD_SKIP_THIS_PUBLIC
+
+#include <fastdds/dds/publisher/qos/DataWriterQos.hpp>
+#include <fastdds/dds/subscriber/qos/DataReaderQos.hpp>
+#include <fastrtps/attributes/PublisherAttributes.h>
+#include <fastrtps/attributes/SubscriberAttributes.h>
+#include <fastrtps/attributes/TopicAttributes.h>
+#include <fastdds/dds/domain/DomainParticipant.hpp>
+#include <fastdds/rtps/participant/RTPSParticipant.h>
```

Comment:
This header is not really used. Instead the one in use is `fastdds/rtps/attributes/RTPSParticipantAttributes.h`

Reply:
Changed in c4a280c1d Refs #15417: #includes.